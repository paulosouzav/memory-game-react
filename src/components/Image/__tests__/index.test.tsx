import React from 'react';
import { mount } from 'enzyme';
import { Image } from '../';

const IMAGE_PROPS = {
  src: 'fake/path/img.png',
  alt: 'Fake Image',
};

describe('Image', () => {
  it('should render without failing', () => {
    const wrapper = mount(<Image {...IMAGE_PROPS} />);
    expect(wrapper).toMatchSnapshot();
  });
});
