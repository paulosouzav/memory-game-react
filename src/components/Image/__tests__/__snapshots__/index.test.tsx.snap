// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image should render without failing 1`] = `
<Image
  alt="Fake Image"
  src="fake/path/img.png"
>
  <img
    alt="Fake Image"
    src="fake/path/img.png"
  />
</Image>
`;
